tests/agents_map.py

```diff
@@ -0,0 +1,60 @@
+"""
+Generate agent parameterization for pytest.
+"""
+import os
+import sys
+from loguru import logger
+from munch import Munch
+from configuration import elasticsearch
+
+sys.path.append(os.path.relpath("../deploy/test-environments/fleet_api/src"))
+from api.agent_policy_api import get_agents  # pylint: disable=wrong-import-position
+
+CIS_AWS_COMPONENT = "cloudbeat/cis_aws"
+CIS_GCP_COMPONENT = "cloudbeat/cis_gcp"
+CIS_AZURE_COMPONENT = "cloudbeat/cis_azure"
+
+
+class AgentExpectedMapping:
+    """
+    This class is used to map expected number of agents that are running each component.
+    """
+
+    def __init__(self):
+        agentless = os.getenv("TEST_AGENTLESS", "false") == "true"
```

Comment:
The same applies here; the processing of environment variables should be centralized in the `configuration.py` file.